src/plugin_features.rs

```diff
@@ -0,0 +1,57 @@
+use std::os::raw::c_char;
+
+pub const CLAP_PLUGIN_FEATURE_INSTRUMENT: *const c_char = "instrument\0".as_ptr() as *const c_char;
+
+pub const CLAP_PLUGIN_FEATURE_AUDIO_EFFECT: *const c_char =
+    "audio-effect\0".as_ptr() as *const c_char;
+
+pub const CLAP_PLUGIN_FEATURE_NOTE_EFFECT: *const c_char =
+    "note-effect\0".as_ptr() as *const c_char;
+
+pub const CLAP_PLUGIN_FEATURE_ANALYZER: *const c_char = "analyzer\0".as_ptr() as *const c_char;
+
+pub const CLAP_PLUGIN_FEATURE_FILTER: *const c_char = "filter\0".as_ptr() as *const c_char;
+pub const CLAP_PLUGIN_FEATURE_PHASER: *const c_char = "phaser\0".as_ptr() as *const c_char;
+pub const CLAP_PLUGIN_FEATURE_EQUALIZER: *const c_char = "equalizer\0".as_ptr() as *const c_char;
+pub const CLAP_PLUGIN_FEATURE_DEESSER: *const c_char = "de-esser\0".as_ptr() as *const c_char;
+pub const CLAP_PLUGIN_FEATURE_PHASE_VOCODER: *const c_char =
```

Comment:
There are extra entries in this list that aren't in 0.26 (compare https://github.com/free-audio/clap/blob/469541c05a0fd59f123422dda3d792ee0f73bc5f/include/clap/plugin-features.h). Not a big deal but may as well make them match.

Reply:
I added them because they were already on the main branch (see the linked diff), but I now moved them to my clap-next branch.